InvokePlaster.ps1

```diff
@@ -129,6 +129,31 @@ function Invoke-Plaster {
         $flags = @{
             DefaultValueStoreDirty = $false
         }
+        $logo = @'
+__________.__                   __
+\______   \  | _____    _______/  |_  ___________
+ |     ___/  | \__  \  /  ___/\   __\/ __ \_  __ \
+ |    |   |  |__/ __ \_\___ \  |  | \  ___/|  | \/
+ |____|   |____(____  /____  > |__|  \___  >__|
+                    \/     \/            \/
+'@, @'
+  ____  _           _
+ |  _ \| | __ _ ___| |_ ___ _ __
+ | |_) | |/ _` / __| __/ _ \ '__|
+ |  __/| | (_| \__ \ ||  __/ |
+ |_|   |_|\__,_|___/\__\___|_|
+'@, @'
+    _____
+   (, /   ) /)
+    _/__ / // _   _  _/_  _  __
+    /     (/_(_(_/_)_(___(/_/ (_
+ ) /
+(_/
+'@
```

Comment:
Found this ASCII art generator - http://www.patorjk.com/software/taag/#p=display&f=Invita&t=Plaster%20